Migrate internal iOS 26 test baselines/expectations to OpenSource.

bls1999 opened the pull request:
#### 1038a2c286392c5bd9dfa51681b5f0684a170151
<pre>
Migrate internal iOS 26 test baselines/expectations to OpenSource.
<a href="https://bugs.webkit.org/show_bug.cgi?id=295607">https://bugs.webkit.org/show_bug.cgi?id=295607</a>
<a href="https://rdar.apple.com/155376303">rdar://155376303</a>

Reviewed by Brianna Fan.

Migrating iOS 26 layout test baselines and expectations from Internal -&gt; OpenSource.

(Truncating file list because I hit the character limit in the PR message.)

* LayoutTests/platform/ios-18/TestExpectations: Added.
* LayoutTests/platform/ios-18/TextExpectations: Removed.
* LayoutTests/platform/ios-18/*: Added new folder for iOS 18-specific baselines.
* LayoutTests/platform/ios/*: Added baselines for iOS 26, moved 18-specific baselines to ios-18.

Canonical link: <a href="https://commits.webkit.org/297148@main">https://commits.webkit.org/297148@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f0b822b0bb795d045eeba7fa8d23e24cc41284ad

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/110695 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/30354 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/20788 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/116722 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/60962 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/31033 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/38943 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/84171 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/113643 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/24789 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/99676 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/64612 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/24155 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/17814 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/60516 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/94178 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/17872 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/119511 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/37735 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/28038 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/93133 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/38109 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/95944 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/92957 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/37987 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/15731 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/33713 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/17858 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/37631 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/37293 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/40632 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/39001 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

briannafan reviewed (approved):
rs=me, let's land after all EWS runs have finished

bls1999 committed:
Migrate internal iOS 26 test baselines/expectations to OpenSource.
https://bugs.webkit.org/show_bug.cgi?id=295607
rdar://155376303

Reviewed by Brianna Fan.

Migrating iOS 26 layout test baselines and expectations from Internal -> OpenSource.

(Truncating file list because I hit the character limit in the PR message.)

* LayoutTests/platform/ios-18/TestExpectations: Added.
* LayoutTests/platform/ios-18/TextExpectations: Removed.
* LayoutTests/platform/ios-18/*: Added new folder for iOS 18-specific baselines.
* LayoutTests/platform/ios/*: Added baselines for iOS 26, moved 18-specific baselines to ios-18.

Canonical link: https://commits.webkit.org/297148@main
